open-vulnerability-clients/src/main/java/io/github/jeremylong/openvulnerability/client/nvd/CvssV3Data.java

```diff
@@ -32,6 +33,7 @@
  * JSON Schema for Common Vulnerability Scoring System version 3.0
  */
 @JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
```suggestion
```

Reply:
These do not declare `additionalProperties: false` in schema, so retained.